screens/leaderboards/Leaderboards.tsx

```diff
@@ -15,9 +15,7 @@ import { containsRef } from '../../xplat/types';
 type LeaderboardTab = 'Monthly' | 'Semesterly';
 type FilterType = 'Anyone' | 'Following';
 
-const Leaderboards = ({
-  navigation,
-}: LeaderboardsScreenProps<'Leaderboards'>) => {
+const Leaderboards = ({}: LeaderboardsScreenProps<'Leaderboards'>) => {
```

Comment:
You can remove the LeaderboardsScreenProps since you dont use the navigation or route object